tests/views/test_collection_exercise.py

```diff
@@ -1257,17 +1261,121 @@ def test_create_collection_events_not_set_sequentially(self, mock_request, mock_
     def test_get_collection_exercises_with_events_and_samples_by_survey_id(self):
         name_space = 'response_operations_ui.controllers.collection_exercise_controllers.'
         with mock.patch(
-                name_space + "get_collection_exercises_by_survey", return_value=self.collection_exercises),\
+                name_space + "get_collection_exercises_by_survey", return_value=self.collection_exercises), \
             mock.patch(
-                name_space + "get_collection_exercise_events_by_id", return_value=self.collection_exercise_events),\
+                name_space + "get_collection_exercise_events_by_id", return_value=self.collection_exercise_events), \
             mock.patch(
-                name_space + "get_linked_sample_summary_id", return_value=self.sample_summary['id']),\
+                name_space + "get_linked_sample_summary_id", return_value=self.sample_summary['id']), \
             mock.patch(
                 name_space + "get_sample_summary", return_value=self.sample_summary):
-
             ce_list = get_collection_exercises_with_events_and_samples_by_survey_id(self.survey['id'])
 
         expected_ce_list = copy.deepcopy(self.collection_exercises)
         expected_ce_list[0]['events'] = self.collection_exercise_events
         expected_ce_list[0]['sample_summary'] = self.sample_summary
         self.assertEqual(ce_list, expected_ce_list)
+
+    @requests_mock.mock()
+    def test_schedule_nude_email_option_not_present(self, mock_request):
```

Comment:
:%s/nude/nudge/g ^^

EDIT:  Fix all of them in this file.

Reply:
fixed